packages/core/test-support/helpers/visit-without-wait.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2019, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+import { registerHelper } from '@ember/test';
+import { run } from '@ember/runloop';
+
+/**
+ * Test helper to visit a route without waiting for
+ * Ref: https://github.com/emberjs/ember.js/blob/v1.12.0/packages/ember-testing/lib/helpers.js#L58
```

Comment:
I don't think this will be long-lived, given the new test helpers. This was what we used in another project and I know it works. I can see if a new version works.

Reply:
3.7.3 worked